doc/user_guide/getting_started.rst

```diff
@@ -221,9 +221,58 @@ in the Signal2D subclass.
     >>> im
     <Signal2D, title: , dimensions: (30|20, 10)>
 
-Note the HyperSpy rearranges the axes position to match the following pattern:
-(navigatons axis 0,..., navigation axis n|signal axis 0,..., signal axis n).
-This is the order used for :ref:`indexing the BaseSignal class <signal.indexing>`.
+Note that HyperSpy rearranges the axes when compared to the array order. The
+following few paragraphs explain how and why it does it.
+
+Depending how the array is arranged, some axes are faster to iterate than
+others. Consider an example of a book as the dataset in question. It is
+trivially simple to look at letters in a line, and then lines down the page,
+and finally pages in the whole book.  However if your words are written
+vertically, it's slightly inconvenient to read top-down (the lines are still
+horizontal, it's just the meaning that's vertical!). It's just awful if
+every letter is on a different page, and for every word you have to turn 5-6
+pages. Exactly the same idea applies here - in order to iterate through the
+data (most often for plotting, but applies for any other operation too), you
+want to keep it ordered for "fast access".
+
+In Python (more explicitly `numpy`) the "fast axes order" is C order (also
+called row-major order). This means that the **last** axis of a numpy array is
+fastest to iterate over (i.e. the lines in the book). An alternative ordering
+convention is the F order (column-major), where it is the reverse - the first
+axis of an array is the fastest to iterate over. In both cases, the further an
+axis is from the `fast axis` the slower it  is to iterate over it. In the book
+analogy you could think, for example, on reading the first lines of all pages,
+then the second and so on.
+
+When data is acquired sequentially it is usually stored in acquisition
+order.When a dataset is loaded, HyperSpy generally stores in memory in the same
+order, which is good for the computer. However, HyperSpy aims at making things
+easy for humans, so it reorders and classify the axes for them.
```

Comment:
"However, HyperSpy will reorder and classify the axes to make it easier for humans"